frontend/src/renderer/components/Charts/LineChart.svelte

```diff
@@ -206,4 +248,12 @@
     border: 0px;
   }
 
+  #spectral-csv-download-button:hover {
+    cursor: pointer;
+  }
+
+  .apexcharts-menu-item.exportCSV {
```

Comment:
Is there a reason why the menu item is not just deleted completely instead of setting display to none?

Reply:
In the "options" object, I think so long as "download" is set to true in "tools:{download:true}" , then all three export options, png, csv, and svg, are given automatically in the toolbar. I did not find a way to pick and choose which export options were available within that "options" object.

Reply:
Yeah, after looking at the docs it seems like it includes all or none.